cmake/compilers/GNU.cmake

```diff
@@ -56,7 +56,7 @@ endif()
 
 # Workaround for heavy tests and too many symbols in debug (rellocation truncated to fit: R_MIPS_CALL16)
 if ("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "mips")
-    set(TBB_TEST_COMPILE_FLAGS ${TBB_TEST_COMPILE_FLAGS} -DTBB_TEST_LOW_WORKLOAD $<$<CONFIG:DEBUG>:-mxgot>)
+    set(TBB_TEST_COMPILE_FLAGS ${TBB_TEST_COMPILE_FLAGS} -DTBB_TEST_LOW_WORKLOAD $<$<CONFIG:DEBUG>:-fPIE -mxgot>)
```

Comment:
According to [MIPS Options (GCC)](https://gcc.gnu.org/onlinedocs/gcc/MIPS-Options.html) "These options (-mxgot, -mno-xgot) have no effect unless GCC is generating position independent code."

Reply:
According to the usage of `-DTBB_TEST_LOW_WORKLOAD` macro there is already a reduction in tests for MIPS. Perhaps, it may be better to reuse it in a more aggressive way even without splitting tests on files and yet test everything we need and still fit in GOT. Have not we considered this option as well?

Reply:
As for -DTBB_TEST_LOW_WORKLOAD approach I guess we just reduce scope of testing on MIPS but the problem is not high workload but a high number of symbols. Splitting testing into separate file doesn't affect test scope at all.